provision/infinispan/Utils.yaml

```diff
@@ -472,10 +471,9 @@ tasks:
           cluster: infinispan
           configMap: crossdc-connect
         EOF
-      - kubectl -n {{ .NAMESPACE }} wait --for=jsonpath='{.status.phase}'=Succeeded Batch/bring-online
-      - defer: |
-          kubectl -n {{ .NAMESPACE }} logs job/bring-online || true
-          kubectl -n {{ .NAMESPACE }} delete Batch/bring-online || true
+      - defer: KUBECONFIG=".task/kubecfg/{{.ROSA_CLUSTER_NAME}}" kubectl -n {{ .NAMESPACE }} delete Batch/bring-online || true
```

Comment:
We're no longer printing the logs in the defer. Won't this mean that if the `kubectl wait` condition fails we'll never get the logs to debug what went wrong?

Reply:
fixed